monkey/infection_monkey/exploit/zerologon.py

```diff
@@ -30,6 +31,13 @@
 
 logger = logging.getLogger(__name__)
 
+ZEROLOGON_EXPLOIT_TAG = "zerologon-exploiter"
+T1003_TAG = "attack-t1003"
+T1098_TAG = "attack-t1098"
+
+
+ZEROLOGON_EVENT_TAGS = {ZEROLOGON_EXPLOIT_TAG, T1003_TAG, T1098_TAG}
```

Comment:
Tags need to be named consistently across files. See https://github.com/guardicore/monkey/blob/eec48e9cd8603313d3d4b489166fe50430e06c1f/monkey/infection_monkey/credential_collectors/ssh_collector/ssh_handler.py#L18..L28.